src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -5,10 +5,11 @@ import Button from "../Button/Button";
 import usePrevious from "../../hooks/usePrevious";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import { baseClassName } from "./ButtonGroupConstants";
+import { ButtonWrapper } from "./ButtonWrapper";
 import "./ButtonGroup.scss";
 
 const ButtonGroup = forwardRef(
-  ({ componentClassName, options, name, disabled, value, onSelect, size, kind, groupAriaLabel }, ref) => {
+  ({ componentClassName, options, name, disabled, value, onSelect, size, kind, groupAriaLabel, tooltipPosition, tooltipHideDelay, tooltipShowDelay, tooltipContainerSelector, tooltipMoveBy}, ref) => {
```

Comment:
I would move all of the tooltip options to the button options to give a bit more flexibility,  this is my opionion